### ISSUE TITLE
Error while installing tox-pipenv

### ISSUE THREAD

**Issue by asyd** [tox-dev/tox-pipenv#41]
Hi, 

I'm trying to install `tox-pipenv` with `pipenv`, however I have the following error:

```
lib/python3.6/site-packages/pipenv/resolver.py", line 71, in main
    sources=project.pipfile_sources,
AttributeError: 'Project' object has no attribute 'pipfile_sources'
```
I must confess I'm not sure it's related to `tox-pipenv` itself or to `pipenv` but `pipenv install` works with other packages.

# Environment
```
pipenv --version
pipenv, version 2018.05.18
```

# Step to reproduces

```
pip install -U --user pipenv
mkdir tmp && cd tmp
pipenv install tox-pipenv
```

**Comment by tonybaloney** [comment]
looks like a versioning problem. I'll repro

**Comment by tonybaloney** [comment]
confirmed this is an issue when using a newer version of pipenv, will fix and patch

**Comment by asyd** [comment]
Thanks!

**Comment by tonybaloney** [comment]
Released in 1.5.0 yesterday
